sql/sql_expression.cc

```diff
@@ -620,6 +620,13 @@ hasUnboundVariables() const
     return false;
 }
 
+bool
+UnboundEntities::
+hasRowFunctions() const
+{
+    return funcs.find("columnCount") != funcs.end();
```

Comment:
Add a comment here saying that we should make this more generic (nobody will remember to add other functions to this list).  Note that rowName(), rowHash(), rowPath(), leftRowHash(), rightRowHash(), etc are also functions that require the row.